src/iperf.h

```diff
@@ -62,6 +62,10 @@
 #include "queue.h"
 #include "cjson.h"
 
+#if defined(HAVE_SSL)
+#include <openssl/bio.h>
```

Comment:
Adding `#include <openssl/evp.h>` here will prevent the compiler from warning about the implicit declaration of [EVP_PKEY_free](https://travis-ci.org/esnet/iperf/jobs/351984688#L574)

Reply:
I agree with this comment.

Reply:
ok, just added. Sorry for the delay